pkg/store/consul/rcstore/fake_store.go

```diff
@@ -62,6 +63,10 @@ func (s *fakeStore) Create(manifest manifest.Manifest, nodeSelector labels.Selec
 	return entry.RC, nil
 }
 
+func (s *fakeStore) CreateTxn(txn *transaction.Tx, manifest manifest.Manifest, nodeSelector labels.Selector, podLabels labels.Set) (fields.RC, error) {
+	panic("transactions not implemented in fake rc store")
```

Comment:
hmmmm, does this mean we don't have test coverage over the Helper functions in rollstore/consul_store.go?